cmake/sysbuild/suit.cmake

```diff
@@ -321,17 +326,30 @@ function(suit_create_package)
     )
   endforeach()
 
-  # First parse which images should be extracted to which cache partition
+  # First parse which images should be extracted to which cache partitions
   set(DFU_CACHE_PARTITIONS_USED "")
+  set(RECOVERY_DFU_CACHE_PARTITIONS_USED "")
```

Comment:
why upper case ?
Is this intended to be used as a global variable / outside of this CMake file ?